motion/ruby_motion_query/subviews.rb

```diff
@@ -63,11 +62,14 @@ def add_subview(view_or_constant, opts={})
         new_view.rmq_appended if appended
 
         if self.stylesheet
-          apply_style_to_view(new_view, style) if style
+          apply_style_to_view(new_view, opts[:style]) if opts[:style]
         end
       end
 
-      RMQ.create_with_array_and_selectors(subviews_added, selectors, @context, self)
+      view = RMQ.create_with_array_and_selectors(subviews_added, selectors, @context, self)
+      opts[:block].call view if opts[:block]
+      opts[:raw_block].call view.get if opts[:raw_block]
+      view
```

Comment:
This is the main place that changed. Note the `raw_block:` vs `block:` above (raw is for the bang methods).